passwords/generate_password.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -e
+
+function check_prereq(){
+    if ! command -v docker --version &> /dev/null
+    then
+        error_exit "docker could not be found."
+    elif [ "$(docker ps | grep -c 'CONTAINER ID')" == 0 ]
+    then
+        error_exit "docker is not running."
+    fi
+}
+
+function gen_password(){
+    env LC_ALL=C tr -dc 'A-Za-z0-9_' < /dev/urandom | head -c "$1"
+}
+
+function sign_enc_keys(){
+    sign=$(sudo docker run circleci/server-keysets:latest generate signing -a stdout)
+    enc=$(sudo docker run circleci/server-keysets:latest generate encryption -a stdout)
```

Comment:
Can you remove sudo from these commands? People can sudo bash the entire script when necessary.

Reply:
👍